contracts/script/utils/ChainIds.sol

```diff
@@ -0,0 +1,11 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.23;
+
+library ChainIds {
+    uint256 public constant MAINNET = 1111; // TBD
```

Comment:
To be consistent, let's call it STORY_MAINNET and set id to 1415

Reply:
Done, thanks